### PR TITLE
[WIP] BitBucket Source example

### PR DESCRIPTION
**Pull request by nachocano**
## Proposed Changes

- Adding an example of a new source: [BitBucket](https://github.com/knative/eventing-sources/pull/231)

### CODE REVIEW

**Comment by nachocano** [comment]
FYI
@grantr 
@Harwayne

File: eventing/sources/sources.yaml [hunk 1]
```diff
@@ -59,6 +59,13 @@ sources:
     description: >
       Registers for events of the specified types on the specified GitLab repository. Brings those
       events into Knative.
+  - name: BitBucket
```
Comment (Harwayne) [inline eventing/sources/sources.yaml]: Follow the instructions in event/sources/README.md, namely

2. Run the generator tool:
    ```shell
    go run eventing/sources/generator/main.go
    ```

**Comment by nachocano** [comment]
/hold
Need to wait until [BitBucket](https://github.com/knative/eventing-sources/pull/231) is approved

File: eventing/sources/sources.yaml (hunk 1 above)
Comment (nachocano) [reply eventing/sources/sources.yaml]: Thanks @Harwayne ! Forgot to run it.

**Comment by nachocano** [comment]
Closing this as I've decided to put it in my own separate repo to minimize maintainability burden.
https://github.com/nachocano/bitbucket-source
Will create a new PR pointing the documentation to that repo

**Pull request closed without merging**